CatalogTests/helpers/driver-manager.ts

```diff
@@ -0,0 +1,19 @@
+import { Browser, Builder, By, WebDriver } from "selenium-webdriver";
```

Comment:
I think that `driver-manager.ts` name for this file does not describe it functionality.

Even if it's a `BasePage`, it's still a page